examples/java/spring-boot/src/main/java/uk/gov/api/springboot/application/NoopRegistryStorage.java

```diff
@@ -6,9 +6,9 @@
 import uk.gov.api.springboot.domain.model.Registry;
 
 @Component
-public class NoopRegistry implements Registry {
+public class NoopRegistryStorage implements Registry.Storage {
```

Comment:
I'd pictured this implementation being implemented in the `Infrastructure` ring alongside the `ApiStorage` implementation, as it'll end up reading from the database?

Reply:
This is still in the same location it was created in, the Application ring. Agreed that the longer term solution would be in the Infrastructure ring. Happy to move it over to follow the `NoopApiStorage` for consistency?

Reply:
sounds good.